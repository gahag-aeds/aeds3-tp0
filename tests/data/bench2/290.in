290 145
63 76 35 21 95 8 18 88 27 60 61 38 73 78 72 17 99 36 58 62 0 82 24 2 79 43 12 55 57 83 20 73 59 7 94 6 15 12 95 94 25 8 85 50 38 57 67 37 93 77 51 46 11 75 48 90 70 12 45 28 47 17 1 59 76 47 65 91 60 12 38 37 20 23 87 10 80 7 99 26 36 2 72 48 78 72 38 0 36 36 80 84 5 33 95 82 81 60 25 93 73 63 30 45 86 17 8 19 76 7 45 13 10 69 61 40 41 51 40 77 87 73 13 45 6 8 27 87 21 52 80 46 68 62 43 54 32 51 25 8 59 70 73 21 39 34 61 32 38 53 10 77 26 75 22 85 36 1 72 57 6 5 55 74 67 98 80 99 50 6 60 61 76 33 82 68 20 95 0 58 0 62 35 79 90 10 64 26 63 88 35 69 93 90 95 61 88 76 12 38 82 72 51 10 6 85 30 78 32 83 36 33 45 23 64 35 85 28 61 49 16 96 18 62 86 14 75 27 42 87 17 76 60 69 38 18 6 69 96 39 52 84 24 49 59 88 37 45 68 98 94 36 47 64 98 85 30 73 64 24 61 82 0 73 3 39 43 9 60 91 0 64 75 24 13 34 64 50 79 84 1 25 21 0 42 19 37 72 93 2 
Sub 102 197
Min 105 200
Add 7 102
Sum 28 123
Sub 124 219
Sum 90 185
Add 65 160
Min 93 188
Add 193 288
Min 135 230
Sub 142 237
Sum 70 165
Add 175 270
Sum 181 276
Sub 50 145
Max 137 232
Add 25 120
Min 99 194
Add 157 252
Max 123 218
Add 123 218
Sum 68 163
Sub 144 239
Max 132 227
Sub 188 283
Max 30 125
Sub 192 287
Min 156 251
Add 61 156
Sum 157 252
Sub 58 153
Sum 162 257
Add 67 162
Sum 192 287
Add 123 218
Sum 124 219
Add 87 182
Sum 122 217
Sub 22 117
Min 19 114
Sub 62 157
Min 163 258
Sub 22 117
Max 43 138
Add 83 178
Min 72 167
Add 113 208
Min 107 202
Sub 104 199
Max 9 104
Add 35 130
Sum 160 255
Add 77 172
Min 113 208
Sub 98 193
Sum 4 99
Sub 76 171
Max 95 190
Add 159 254
Min 70 165
Add 185 280
Min 150 245
Add 37 132
Min 57 152
Add 81 176
Min 94 189
Sub 180 275
Sum 101 196
Add 21 116
Max 7 102
Sub 120 215
Max 82 177
Add 103 198
Min 141 236
Sub 58 153
Sum 185 280
Sub 146 241
Max 104 199
Add 31 126
Sum 55 150
Add 113 208
Min 194 289
Add 149 244
Sum 123 218
Sub 112 207
Max 52 147
Sub 126 221
Sum 187 282
Sub 80 175
Max 91 186
Add 63 158
Max 70 165
Sub 174 269
Min 33 128
Sub 60 155
Min 61 156
Sub 60 155
Max 45 140
Add 161 256
Min 80 175
Add 179 274
Min 20 115
Sub 96 191
Sum 87 182
Add 95 190
Max 87 182
Add 11 106
Min 46 141
Add 191 286
Min 169 264
Add 35 130
Max 43 138
Sub 168 263
Sum 117 212
Add 165 260
Max 19 114
Sub 168 263
Sum 30 125
Sub 140 235
Max 181 276
Sub 54 149
Min 136 231
Sub 184 279
Max 34 129
Sub 102 197
Sum 49 144
Sub 28 123
Min 162 257
Add 27 122
Sum 122 217
Add 47 142
Max 11 106
Sub 76 171
Max 76 171
Add 31 126
Min 170 265
Add 163 258
Max 169 264
Add 149 244
Max 93 188
Add 77 172
Min 117 212
Add 135 230
Max 179 274
Add 39 134
