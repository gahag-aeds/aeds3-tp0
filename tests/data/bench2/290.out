-1
4738
4600
-2
-2
4396
5124
97
-2
97
4439
97
101
-1
5072
5230
4879
4860
4922
1
1
99
2
2
100
5288
1
4806
99
3
1
1
1
4918
102
101
0
5194
100
4775
-1
5002
102
5087
100
103
3
2
101
3
1
4578
100
2
0
103
4881
103
5022
101
-2
102
4725
-3
4610
103
102
-3
101
102
-1
101
